test/test_proxy_tensor.py

```diff
@@ -773,6 +768,7 @@ def f(a, b):
     xfail('msort', ''),  # aten.sort.default - couldn't find symbolic meta function/decomposition
     xfail('mv', ''),  # aten.mv.default - couldn't find symbolic meta function/decomposition
     xfail('nanmean', ''),  # The underlying op of 'aten.stride' has no overload name '_schema'
+    xfail('nanquantile', ''),  # Could not run 'aten::equal' with arguments from the 'Meta' backend.
```

Comment:
This is a regression, right? How difficult is it to write a meta rule for this? The rule should be that that aten::equal always returns a boolean.

Reply:
There's some python API for registering a meta kernel at https://github.com/pytorch/pytorch/blob/master/torch/_meta_registrations.py

Reply:
This isn't a regression. Previously it used to fail all the 3 tests in the suite. Now it fails only one of them.

On master it is part of `make_fx_failures`.
https://github.com/pytorch/pytorch/blob/18db7f36d785f245f3fa2084d809a2c7f1b82994/test/test_proxy_tensor.py#L606-L608

However, in this PR, we move it to `symbolic_tensor_failures `.

Ref to the skips used by the tests.
https://github.com/pytorch/pytorch/blob/18db7f36d785f245f3fa2084d809a2c7f1b82994/test/test_proxy_tensor.py#L1037-L1054

Reply:
> The rule should be that that aten::equal always returns a boolean.

Regarding this what should it return? I don't think it can return concrete values like `True/False`.

Reply:
Oh I see, the separate failure lists confused me

Reply:
Let's punt the meta implementation since it is orthogonal, we don't know how to get it to work, and this PR doesn't regress the current state.

Reply:
To make the meta work, need to stop using aten::equal. An asynchronous assert would do the trick.